full-service/src/json_rpc/v2/api/wallet.rs

```diff
@@ -1443,8 +1444,17 @@ where
                 .map_err(format_error)?;
             JsonCommandResponse::validate_sender_memo { validated: result }
         }
-        JsonCommandRequest::verify_address { address } => JsonCommandResponse::verify_address {
-            verified: service.verify_address(&address).is_ok(),
+        JsonCommandRequest::verify_address { address } => match service.verify_address(&address) {
```

Comment:
:question: where does this, https://mobilecoin.gitbook.io/full-service-api/api-endpoints/v2/account/address/verify_address, bet updated at?

Reply:
We maintain the docs using gitbook itself.  [Here is a preview of the edits ](https://mobilecoin.gitbook.io/full-service-api/~/changes/AqCyVo43Lcl820LFbEoI/api-endpoints/v2/account/address/verify_address) that correspond to this PR